php/libraries/NDB_Factory.class.inc

```diff
@@ -25,6 +25,11 @@ use \LORIS\StudyEntities\Candidate\CandID;
  * @author   Dave MacFarlane <david.macfarlane2@mcgill.ca>
  * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
  * @link     https://www.github.com/aces/Loris/
+ *
+ * Phan currently has bad support for PHPUnit's Mock objects which is
+ * creating false positive with the below rule.
+ *
+ * @phan-file-suppress PhanUndeclaredClassMethod
```

Comment:
I think this is a rare reasonable use of suppressing the rule.. but at some point we should probably fix the code to not be so intertwined with PHPUnit.